src/modal/modal.scss

```diff
@@ -74,3 +82,34 @@
     transition-delay: 0s;
   }
 }
+
+@mixin iui-modal-full-page {
+  --iui-full-page-modal-transform-x: 100%;
+  --iui-full-page-modal-transform-y: 0%;
+
+  visibility: visible;
+  opacity: 1;
+  transform: translate(var(--iui-full-page-modal-transform-x), var(--iui-full-page-modal-transform-y));
+  @media (prefers-reduced-motion: no-preference) {
+    transition: transform $iui-speed ease-in;
+  }
+
+  > .iui-modal-dialog {
+    height: 100vh;
+    width: 100vw;
+    left: 0;
+    top: 0;
+    transform: none;
+    max-width: initial;
+    min-width: initial;
+    border-radius: 0;
+    box-shadow: none;
+  }
+
+  &.iui-modal-visible {
+    transform: translate(0, 0);
+    @media (prefers-reduced-motion: no-preference) {
```

Comment:
If both transitions are using the same ease value, this should not be necessary.

Reply:
They are not using the same ease value. I am following [the suggestions made here](https://css-tricks.com/ease-out-in-ease-in-out/#:~:text=There%20are%20two,and%20without%20delay.) for moving on and off the screen.